huaweicloud/services/mrs/resource_huaweicloud_mapreduce_cluster.go

```diff
@@ -252,6 +252,12 @@ func ResourceMRSClusterV2() *schema.Resource {
 				MaxItems: 1,
 				Elem:     smnNotifySchema(),
 			},
+			"mrs_ecs_default_agency": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
```

Comment:
```suggestion
				Optional:    true,
				Computed:    true,
				ForceNew:    true,
```
Supplement a computed behavior to prevent changes while some regions return a default agency and the script does not supported.

Reply:
done